Makefile

```diff
@@ -229,8 +229,8 @@ build-local-ubuntu-all: build-local-linux  ## Build kusion & kcl tool chain for
 		./cmd/kcl-go
 
 	# Build kcl-openapi
-	# cd ./cmd/kcl-openapi && GOOS=linux GOARCH=amd64 CGO_ENABLED=0 \
-	# 	go build -o ${PWD}/_build/bundles/kusion-ubuntu/bin/kcl-openapi
+	cd ./cmd/kcl-openapi && GOOS=linux GOARCH=amd64 CGO_ENABLED=0 \
+		go build -o ${PWD}/_build/bundles/kusion-ubuntu/bin/kcl-openapi
 
 	# chmod +x
 	-chmod +x ./_build/bundles/kusion-linux/bin/kusion
```

Comment:
```-chmod +x ./_build/bundles/kusion-linux/bin/kusion```
the script seems wrong since the binaries are under `kusion-ubuntu` directory instead of `kusion-linux`

Reply:
> -chmod +x ./_build/bundles/kusion-li

fixed.